test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -989,6 +989,7 @@ def _revert_pbsconf_comm(self, primary_server, vals_to_set):
                 self.du.set_pbs_config(comm.hostname, confs=new_pbsconf)
                 comm.pbs_conf = new_pbsconf
                 comm.pi.initd(comm.hostname, "restart", daemon="comm")
+                comm.isUp()
```

Comment:
I think you should do an assertTrue to make sure that the daemon is up, same comment for the rest of the daemons.